ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -46,20 +57,33 @@ async function buildWebDriverAgent (xcodeVersion) {
     '--exclude', 'lib',
     '--exclude', 'test',
     '--exclude', 'bundles',
+    '--exclude', 'azure-templates',
   ], {cwd: rootDir});
 
-  // Moved DerivedData/WebDriverAgent-* from Library to uncompressed folder
-  const derivedDataPath = path.resolve(os.homedir(), 'Library', 'Developer', 'Xcode', 'DerivedData');
+  // Move DerivedData/WebDriverAgent-* from Library to "uncompressed" folder
   const wdaPath = (await fs.glob(`${derivedDataPath}/WebDriverAgent-*`))[0];
```

Comment:
Please use the same approch with `cwd` and `absolute` here. Also, it makes sense to verify the resulting array has at least one item